packages/message-box-react/integration/message-box.spec.js

```diff
@@ -0,0 +1,26 @@
+/// <reference types="cypress" />
+
+context("MessageBox", () => {
+    beforeEach(() => {
+        cy.testComponent("messagebox");
+    });
+
+    it("renders correctly", () => {
+        cy.getComponent().toMatchImageSnapshot();
+        cy.setChoice("Type", "Suksessmelding").getComponent().toMatchImageSnapshot();
+        cy.setChoice("Type", "Advarselsmelding").getComponent().toMatchImageSnapshot();
+        cy.setChoice("Type", "Feilmelding").getComponent().toMatchImageSnapshot();
+
+        cy.setDarkMode();
+
+        cy.getComponent().toMatchImageSnapshot();
+        cy.setChoice("Type", "Advarselsmelding").getComponent().toMatchImageSnapshot();
+        cy.setChoice("Type", "Suksessmelding").getComponent().toMatchImageSnapshot();
+        cy.setChoice("Type", "Infomelding").getComponent().toMatchImageSnapshot();
```

Comment:
Er det meninga at infomelding ikke testes i light-mode og at feilmelding ikke testes i darkmode? 
Hadde likt at rekkefølgen for light og dark mode var lik

Reply:
Begge testes begge steder :) Siden starter med Infomelding valgt, så det som skjer her er at Cypress klikker seg gjennom de andre valgene, så setter på dark mode, og så klikker seg tilbake igjen i listen